### PR TITLE
get rid of try/catch to find one specific error.

### PR DESCRIPTION
**Pull request by kdaily**
Just append errors to be handled by the warning and error log handler.

This is a starting point - I think that the `FileTypeFormat` could return `(valid, message, warning)` to simplify further.

### CODE REVIEW

**Comment by thomasyu888** [comment]
Also, there are a couple classes that throw ValueErrors in `_get_dataframe`.  These errors would have to be caught as well and appeneded.

File: genie/input_to_database.py [hunk 1]
```diff
@@ -166,21 +166,16 @@ def _check_valid(syn, filepaths, center, filetype, filenames,
...
-        valid = False
+    valid, message, filetype = validate.validate_single_file(
+        syn,
+        filepaths,
+        center,
+        oncotreelink=oncotree_link,
+        testing=testing)
+    logger.info("VALIDATION COMPLETE")
+
+    valid, message = determine_validity_and_log(total_error, warning)
```
Comment (thomasyu888) [inline genie/input_to_database.py]: This is unnecessary as it is already done in `validate_single_file`

**Review by thomasyu888 — CHANGES_REQUESTED** [review]

**Comment by thomasyu888** [comment]
Will also have to change tests in `test_input_to_database.py`: `test_filetype_validate_single_file`, and `test_wrongfiletype_validate_single_file` and `ValueErrors` are no longer thrown.  Will have to check for `total_error`

File: genie/input_to_database.py (hunk 1 above)
Comment (kdaily) [reply genie/input_to_database.py]: removed function as it isn't doing anything extra for validate single file

**Review by thomasyu888 — APPROVED** [review]

**Pull request merged**